### PR TITLE
feat: Add custom_property events

### PR DESCRIPTION
**Pull request by acouvreur**
custom_property and custom_property_values events are added.

Closes https://github.com/google/go-github/issues/3207

- [x] I have run `script/fmt.sh`
- [x] I have run `script/test.sh`
- [x] I have run `script/lint.sh`
- [x] I have run `script/generate.sh` because changes were detected

### CODE REVIEW

File: github/event_types.go [hunk 1]
```diff
@@ -132,6 +132,43 @@ type CreateEvent struct {
...
+
+	// The following fields are only populated by Webhook events.
+	Enterprise   *Enterprise   `json:"enterprise,omitempty"`
+	Installation *Installation `json:"installation,omitempty"`
+	Org          *Organization `json:"organization,omitempty"`
+	Sender       *User         `json:"sender,omitempty"`
+}
+
+// CustomPropertyValuesEvent represents an update to a custom property.
+// The Webhook event  name is "custom_property_values"
```
Comment (gmlewis) [inline github/event_types.go:155]:
```suggestion
// The Webhook event name is "custom_property_values".
```

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @acouvreur!
One minor tweak, please, then we should be ready for a second LGTM+Approval from any other contributor to this repo before merging.

**Comment by acouvreur** [comment]
> Thank you, @acouvreur! One minor tweak, please, then we should be ready for a second LGTM+Approval from any other contributor to this repo before merging.

Done!

**Comment by gmlewis** [comment]
Thank you, @stevehipwell !
Merging.

**Pull request merged**